test/input_gen/gen_layer_tests.py

```diff
@@ -603,14 +845,21 @@ def call(self, inputs):
     record_single_fp16(conv, (3, 1, 1, 4), "conv1d_mb_causal_dilation_w16a16")
 
     concat = K.layers.Concatenate(axis=3)
-    record_single_fp16(concat, [(2,3,3,2), (2, 3, 3, 3)], "concat_dim3_w16a16")
+    record_single_fp16(concat, [(2, 3, 3, 2), (2, 3, 3, 3)], "concat_dim3_w16a16")
 
     concat = K.layers.Concatenate(axis=2)
-    record_single_fp16(concat, [(2,3,2,3), (2, 3, 3, 3)], "concat_dim2_w16a16")
+    record_single_fp16(concat, [(2, 3, 2, 3), (2, 3, 3, 3)], "concat_dim2_w16a16")
 
     concat = K.layers.Concatenate(axis=1)
-    record_single_fp16(concat, [(2,2,3,3), (2, 3, 3, 3)], "concat_dim1_w16a16")
+    record_single_fp16(concat, [(2, 2, 3, 3), (2, 3, 3, 3)], "concat_dim1_w16a16")
 
     positional_encoding = PositionalEncoding(10, 6)
-    record_single_fp16(positional_encoding, [(3, 1, 7, 6)], "positional_encoding_partial_w16a16")
-    record_single_fp16(positional_encoding, [(3, 1, 10, 6)], "positional_encoding_w16a16")
\ No newline at end of file
+    record_single_fp16(
+        positional_encoding, [(3, 1, 7, 6)], "positional_encoding_partial_w16a16"
+    )
+    record_single_fp16(
+        positional_encoding, [(3, 1, 10, 6)], "positional_encoding_w16a16"
+    )
+
+    added = K.layers.Add()
+    record_single_fp16(added, [(2, 3, 3, 3), (2, 3, 3, 3)], "added_w16a16")
```

Comment:
I wonder if the torch do the overflow check. if the addition result is over the range of float16, how it can handle?

Reply:
I will check further about this issue